os/net/bluetooth/bt_hcicore.h

```diff
@@ -450,4 +473,14 @@ int hci_le_create_conn(FAR const bt_addr_le_t *addr);
 
 void cmd_queue_init(void);
 
+void hci_disconn_complete_internal(FAR struct bt_buf_s *buf);
+
```

Comment:
Please remove these extra lines.

Reply:
The blank line is required for readability of the multiple functions